modules/ROOT/pages/to-find-info.adoc

```diff
@@ -4,336 +4,120 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Anypoint Exchange search looks for terms in:
-
-. Each asset’s name
-. Tags
-. Categories
-. Custom attributes (also known as custom fields)
-. The content of the asset portal pages
-. Properties defined in RAML and OAS API specifications
+The Exchange catalog of assets can display hundreds of assets that help developers discover assets to create projects quickly and drive adoption of best practices and reuse. You can find assets from Mulesoft's ecosystem or your own organization. Based on your permission group, you can search multiple business groups, documentation, tags, custom attributes, and details from API specifications. 
 
-In an asset portal, click a tag or category element to create a filtered search showing similar assets.
+To help find assets, searches can be refined using search terms like a name of an API, by filters like organizations, categories, tags, and lifecycle states, or by ranking.
 
-The original Exchange search experience is available in Anypoint Platform Private Cloud Edition, MuleSoft Government Cloud, the US cloud, and the EU cloud. The new Exchange search experience is available in the US cloud and the EU cloud. The new search includes more options for search ranking, and more options and a better interface for searching by types, tags, categories, organizations, and lifecycle states.
+Anypoint Exchange search looks for terms in:
```

Comment:
This is already covered in the "Search by Keyword" section, I think it's redundant to keep it here, too.
I also think this list is more accurate than the one in "Search by Keyword" section, so let's replace it.